### PR TITLE
Sync downstream/master with upstream v1.9 changes

### PR DESCRIPTION
**Pull request by atheo89**
Related PR: https://issues.redhat.com/browse/RHOAIENG-13494

Steps for the sync: 
```
git checkout -b RHOAIENG-13494 downstream/sync-master 
git pull  upstream v1.9-branch --no-rebase
fix conflicts
git add . 
git commit
git push 
```

### CODE REVIEW

File: components/notebook-controller/config/overlays/service-mesh/OWNERS [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Each list is sorted alphabetically, additions should maintain that order
+approvers:
+- aslakknutsen
+- bartoszmajsak
+- cam-garrison
+
+reviewers:
+- aslakknutsen
+- bartoszmajsak
+- cam-garrison
```
Comment (harshad16) [inline components/notebook-controller/config/overlays/service-mesh/OWNERS:10]:
```suggestion
approvers: []
# - aslakknutsen
# - bartoszmajsak
# - cam-garrison

reviewers: []
# - aslakknutsen
# - bartoszmajsak
# - cam-garrison
```
Till we find when these contributor would be added to RHOAI, lets keep this empty.

**Review by harshad16 — CHANGES_REQUESTED** [review]
Lets removes owner from sevrice-mesh till , we get more information

File: components/notebook-controller/config/overlays/service-mesh/OWNERS (hunk 1 above)
Comment (atheo89) [reply components/notebook-controller/config/overlays/service-mesh/OWNERS]: done

**Comment by jiridanek** [comment]
/lgtm

the static checks lints fail when there's new code coming in, it's expected

**Review by harshad16 — APPROVED** [review]
/lgtm
/approve

The manifests of the notebook-controller and odh-notebook-controller, seems to be only including service mesh changes, which are in different overlays.

**Comment by harshad16** [comment]
Merging to make the master branch ready.

**Pull request merged**